NEWS.md

```diff
@@ -108,6 +117,12 @@
   the first argument of an S3 method (which is necessarily evaluated
   in order to detect which class to dispatch to). (#295, #300).
 
+* Parentheses around `!!` are automatically removed. This is useful
```

Comment:
I think you need to somehow mention that removing the automatic parens removal will not change precedence, and parens will be inserted when printing if needed

``` r
library(rlang)
expr(-!!expr(x+1))
#> -(x + 1)
```

Reply:
"is useful" -> "makes the generated call cleaner"